setup.py

```diff
@@ -15,7 +15,7 @@
     url="https://github.com/modin-project/modin",
     long_description=long_description,
     long_description_content_type="text/markdown",
-    install_requires=["pandas==0.23.4", "ray==0.6.2", "numpy<=1.15.0"],
+    install_requires=["pandas==0.23.4", "ray==0.6.2", "numpy<=1.15.0", "sqlalchemy>=1.2.17"],
```

Comment:
I don't think we need `sqlalchemy` in `install_requires`. Users can have this installed if they wish to use it (similar to other file formats).

Reply:
Good point! Fixed!